web-console/future-openshift/diagnostics/diagnostics.md

```diff
@@ -0,0 +1,35 @@
+# Downloading Diagnostics
+
+Users can download a diagnostic report of their system and send it to Red Hat from the console.
+
+
+## Help menu
+![dropdown](img/dropdown.png)
+* The "Download Diagnostics" option is available from the help menu.
```

Comment:
"Download diagnostics" sounds like I'm going to get some information about downloads. I'd suggest "Diagnostic report" or something similar. The other items in that menu are nouns, not verbs/actions, for what that's worth.

Reply:
Good point! Will update.

Reply:
"Diagnostic report" does not seem indicative of what the user is attempting to do which is to "Generate diagnostics" (to download, to report to RH etc.).